packages/js/product-editor/src/blocks/product-fields/regular-price/edit.tsx

```diff
@@ -96,34 +142,152 @@ export function Edit( {
 	}, [] );
 
 	return (
-		<div { ...blockProps }>
-			<BaseControl
-				id={ regularPriceId }
-				help={
-					regularPriceValidationError
-						? regularPriceValidationError
-						: renderHelp()
-				}
-				className={ classNames( {
-					'has-error': regularPriceValidationError,
-				} ) }
-			>
-				<InputControl
-					{ ...inputProps }
+		<>
+			<div { ...blockProps }>
+				<BaseControl
 					id={ regularPriceId }
-					name={ 'regular_price' }
-					ref={ regularPriceRef }
-					label={
-						tooltip ? (
-							<Label label={ label } tooltip={ tooltip } />
-						) : (
-							label
-						)
+					help={
+						regularPriceValidationError
+							? regularPriceValidationError
+							: renderHelp()
 					}
-					disabled={ disabled }
-					onBlur={ validateRegularPrice }
-				/>
-			</BaseControl>
-		</div>
+					className={ classNames( {
+						'has-error': regularPriceValidationError,
+					} ) }
+				>
+					<InputControl
+						{ ...inputProps }
+						id={ regularPriceId }
+						name={ 'regular_price' }
+						ref={ regularPriceRef }
+						label={
+							tooltip ? (
+								<Label label={ label } tooltip={ tooltip } />
+							) : (
+								label
+							)
+						}
+						disabled={ disabled }
+						onBlur={ validateRegularPrice }
+					/>
+				</BaseControl>
+			</div>
+			<VariationQuickUpdateFill name="list-price">
+				{ ( { selection, onChange, onClose } ) => {
+					const ids = selection.map( ( { id } ) => id );
+					return (
+						<MenuGroup label={ __( 'List price', 'woocommerce' ) }>
```

Comment:
It would be good to have a `VariationQuickUpdateMenuGroup` slot fill instead, to help enforce and encapsulate the `MenuGroup` usage.

The `VariationQuickUpdateMenuGroup` could then have a slot for the existing `VariationQuickUpdateMenuItem` slot fill.